server/settings.py

```diff
@@ -197,9 +197,10 @@
 
 
 # Files in this directory will be served by WhiteNoise at the site root.
-WHITENOISE_ROOT = os.path.join(ROOT, "build")
-STATIC_ROOT = os.path.join(ROOT, "static")
+WHITENOISE_ROOT = os.path.join(BASE_DIR, "build")
+STATIC_ROOT = os.path.join(BASE_DIR, "server")
 STATIC_URL = EVAL_FRAME_ORIGIN
+STATICFILES_DIRS = (os.path.join(STATIC_ROOT, "static"),)
```

Comment:
Then change this to STATICFILES_DIRS = (os.path.join(BASE_DIR, "server", "static"),)

Reply:
when I do this, I get this error: 
```
server_1  | ?: (staticfiles.E002) The STATICFILES_DIRS setting should not contain the STATIC_ROOT setting.
```